.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+on: push
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
```

Comment:
`jobs` is where we define all the separate flows that run as a part of this workflow. A single workflow can have multiple jobs because you might want to only run more expensive end-to-end tests on your `main` branch or have some jobs run concurrently.

`ci` here is the identifier for this specific job. If we had other jobs we could do things like 'only run this job if `ci` succeeded'.

`runs-on` tells GitHub Actions which OS we need to run this job on. I've not had to use anything other than `ubuntu-latest` so far but who knows 🤷.